### DISCUSSION TITLE
Any way to lock a cursor in a window?

### DISCUSSION THREAD

**Discussion by yorunoken** [hyprwm/Hyprland#7305] [Q&A]
Hello, I play osu! with raw input enabled at very low sensitivity, this causes the normal cursor which is invisible to go out of the game window before the game cursor reaches halfway.
The game has built-in cursor confinement, but it doesn't work in Hyprland. Is there a way to do it using Hyprland?

**Comment by shogeki** [comment]
you can try gamescope with --force-grab-cursor argument

or something like this as a last resort
```
monitor=desc:1, 1920x1080@60, 0x0,1
monitor=desc:2, 1920x1080@60, 10000x0,1
monitor=desc:3, 1920x1080@60, 20000x0,1
```
